training/distributed_training/pytorch/model_parallel/gpt-j/11_train_gptj_smp_tensor_parallel_notebook.ipynb

```diff
@@ -0,0 +1,988 @@
+{
```

Comment:
<p>Recommended to provide an explanation/link for how to request a quota limit, since both ml.p4d.24xlarge and ml.p3.16xlarge have a default quota of 0 instances.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>